README.md

```diff
@@ -389,11 +450,11 @@ Sample response:
 }
 ```
 
-### Invoke-connection
+### InvokeConnection
 
 Using  InvokeConnection, end-user applications can integrate checkout/card issuance flow with their apps/systems. To invoke connection, use the invokeConnection(config ConnectionConfig) method of the Skyflow client. The config object must have `connectionURL`,`methodName` and remaining are optional. 
 
-Using the InvokeConnection method, you can integrate their server-side application with third party APIs and services without directly handling sensitive data. Prior to invoking the `InvokeConnection` method, you must have created a connection and have a connectionURL already generated. Once you have the connectionURL, you can invoke a connection by using the **invokeConnection(config ConnectionConfig)** method. The config parameter must include a `connectionURL` and `methodName`. The other fields are optional.
+Using the InvokeConnection method, you can integrate their server-side application with third party APIs and services without directly handling sensitive data. Prior to invoking the `InvokeConnection` method, you must have created a connection and have a connectionURL already generated. Once you have the connectionURL, you can invoke a connection by using the **InvokeConnection(config ConnectionConfig)** method. The config parameter must include a `connectionURL` and `methodName`. The other fields are optional.
```

Comment:
Who is "their" on line 457?
I would update the following sentences to read: 

The InvokeConnection method lets you bypass handling sensitive data by integrating (replace their) server-side application with third-party APIs.

Prior to invoking the `InvokeConnection` method, you must create a connection and generate a connectionURL.

Reply:
"their" refers to any third-party services